include/service/entities/content/article_service_v2.js

```diff
@@ -145,23 +145,54 @@ module.exports = function(pb) {
      * @param {String} id
      * @param {object} options
      * @param {Boolean} [options.render=false]
+     * @param {Boolean} [options.readMore=false]
      * @param {Function} cb
      */
     ArticleServiceV2.prototype.get = function(id, options, cb) {
-        
         var self = this;
+        var renderOptions = {};
+        if (!options || options.readMore === undefined) {
+            renderOptions.readMore = false;
+        }
+        else {
+            renderOptions.readMore = options.readMore;
+        }
+
         var afterGet = function(err, article) {
             if (util.isError(err) || article === null || !options || !options.render) {
                 return cb(err, article);
             }
             
             //complete the rendering
-            self.render([article], function(err/*, articles*/) {
+            self.render([article], renderOptions, function(err/*, articles*/) {
```

Comment:
One small nit-picky item.  Would you mind moving the creation of the ```renderOptions``` block here?  It marries the logic the options to the code that utilizes it.